src/autorest-core/lib/pipeline/plugins/autorest-dotnet.ts

```diff
@@ -3,10 +3,129 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
this file was merged in a way that undos previous work - it's really supposed to look like https://github.com/Azure/autorest/blob/master/src/autorest-core/lib/pipeline/plugins/autorest-dotnet.ts

specifically, the class `AutoRestDotNetPlugin` is not longer used or required.